engine/api/integration/dao_project_integration.go

```diff
@@ -0,0 +1,187 @@
+package integration
+
+import (
+	"context"
+
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/engine/api/database/gorpmapping"
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/log"
+)
+
+// DeleteIntegration deletes a integration
+func DeleteIntegration(db gorp.SqlExecutor, integration sdk.ProjectIntegration) error {
+	pp := dbProjectIntegration{ProjectIntegration: integration}
+	if _, err := db.Delete(&pp); err != nil {
+		return sdk.WrapError(err, "Cannot remove integration")
+	}
+	return nil
+}
+
+func load(db gorp.SqlExecutor, query gorpmapping.Query, clearPassword bool) (sdk.ProjectIntegration, error) {
```

Comment:
Can you create a dedicated func to load with and without decrypt like for application dao instead of the boolean ?